pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -283,6 +282,15 @@ func (c *EtcdCertSignerController) syncAllMasterCertificates(ctx context.Context
 		Type: corev1.SecretTypeOpaque,
 		Data: allCerts,
 	}
+
+	safe, err := c.quorumChecker.IsSafeToUpdateRevision()
```

Comment:
please also leave a comment here on why we are checking that again

Reply:
make update-gofmt  , done